client/blocks/reader-full-post/_content.scss

```diff
@@ -213,21 +216,42 @@
 	img:first-child {
 		margin-top: 0;
 	}
+
+	// Gutenberg blocks
+	.wp-block-image,
+	.wp-block-embed {
+		margin-bottom: 1em;
+	}
+
+	ul.wp-block-gallery {
+		list-style-type: none;
+		margin-left: 0;
+		margin-bottom: 1em;
+	}
+
+	.wp-block-embed .embed-vimeo {
+		padding-top: 0;
+		margin-bottom: 1.4em;
+	}
+
+	.wp-block-cover-image {
```

Comment:
Hide cover image for the moment (the background image is stripped by post normalizer).